src/navigation/components/Navigation.jsx

```diff
@@ -1,56 +1,60 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
+import Routes from './Routes.jsx';
 
 class Navigation extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { location: "/" }
+    }
+
+    handleLocation(locationParam) {
```

Comment:
Location parameters have a different connotation in web applications. `locationParam` should simply be called `location`. That way you have:
```javascript
this.setState({ location });
```
Try to avoid abbrev var names. 😛

Also, `handleLocation` is not a good method name. Handle location what? Location param? Location change? Location param update?